htdocs/product/fournisseurs.php

```diff
@@ -810,6 +863,16 @@ function update_price_from_multicurrency() {
                     print_liste_field_titre("BarcodeType", $_SERVER["PHP_SELF"], "pfp.fk_barcode_type", "", $param, '', $sortfield, $sortorder, 'center ');
                 }
 				print_liste_field_titre("DateModification", $_SERVER["PHP_SELF"], "pfp.tms", "", $param, '', $sortfield, $sortorder, 'right ');
+
+				// fetch optionals attributes and labels
+				$extralabels=$extrafields->fetch_name_optionals_label("product_fournisseur_price");
+				foreach ($extralabels as $key => $value) {
```

Comment:
$extralabels is deprecated.
You must use $extrafields->attributes["product_fournisseur_price"]['label'] instead.